src/jenkins_job_insight/main.py

```
@@ -1353,6 +1353,18 @@ async def _safe_update_progress_phase(phase: str) -> None:
                     exc_info=True,
                 )
 
+        # Auto-assign job metadata from name pattern rules
+        try:
+            await storage.auto_assign_job_metadata(
+                body.job_name, settings.metadata_rules
+            )
+        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)